pkg/util/kubernetes/autoscalers/processor_test.go

```diff
@@ -14,17 +14,20 @@ import (
 	"testing"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/clusteragent/custommetrics"
+
+	"github.com/prometheus/client_golang/prometheus"
+	promutil "github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/zorkian/go-datadog-api.v2"
 	autoscalingv2 "k8s.io/api/autoscaling/v2beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/DataDog/datadog-agent/pkg/clusteragent/custommetrics"
-	"github.com/stretchr/testify/assert"
 )
 
 type fakeDatadogClient struct {
```

Comment:
I see two `fakeDatadogClient` (in 2 different packages) do you think it is possible to have only one?

maybethe best would be having this fake client in the `gopkg.in/zorkian/go-datadog-api.v2` package.

Reply:
I can do that in another PR, you are right